lib/src/replay/screenshot/screenshot_capturer.dart

```diff
@@ -30,8 +34,8 @@ class ViewTreeSnapshotStatus {
 class ScreenshotCapturer {
```

Comment:
what do you think about changing the class name? `ScreenshotCapturer` looks weird to me

Reply:
find to rename it, its only used internally